src/com/reco1l/osu/data/Scores.kt

```diff
@@ -26,10 +27,14 @@ data class ScoreInfo @JvmOverloads constructor(
     val id: Long = 0,
 
     /**
-     * The beatmap path.
-     * This is the relative path to [Config.getBeatmapPath()].
+     * The beatmap filename.
      */
-    val beatmapPath: String,
+    val beatmapFilename: String,
```

Comment:
Index needs to be updated to account for this. Something like the following should do:
```kt
Index(name = "beatmapIdx", value = ["beatmapFile", "beatmapSetDirectory"])
```

Reply:
Solved with [b6466f2](https://github.com/osudroid/osu-droid/pull/394/commits/b6466f29563206579631a28f07eb75d7a848ec22)